network/transport/v2/protocol.go

```diff
@@ -0,0 +1,111 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"github.com/nuts-foundation/nuts-node/core"
+	"github.com/nuts-foundation/nuts-node/network/log"
+	"github.com/nuts-foundation/nuts-node/network/transport"
+	"github.com/nuts-foundation/nuts-node/network/transport/grpc"
+	grpcLib "google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// New creates an instance of the v2 protocol.
+func New() transport.Protocol {
+	return &protocol{}
+}
+
+type protocol struct {
```

Comment:
I agree with the `v2/protocol` naming, but for v1 it's still `v1/ protocolV1`.....

Reply:
For the sake of small PRs, can I (or someone else) fix that in a subsequent PR?

(made an issue: https://github.com/nuts-foundation/nuts-node/issues/628)